mne/epochs.py

```diff
@@ -183,6 +184,93 @@ def _save_split(epochs, fname, part_idx, n_parts, fmt):
     end_file(fid)
 
 
+def _handle_duplicate_events(events, event_id, event_repeated):
```

Comment:
i like `event_repeated_policy` better.

Reply:
I'd rather not make the name longer. There is a tradeoff between explicit name clarity and usability. There is a good reason we don't go farther down the road of making it even more explicit like `how_to_behave_when_events_occur_on_the_same_sample`. I think `event_repeated` is clear enough as it is and more in line with how we tend to balance clarity with brevity.

Reply:
I'll stay with `event_repeated` then, but will rename `_handle_duplicate_events` -> `_handle_event_repeated`

why don't you like "handle" @massich ? We are "handling" a situation after all?